docs/ecflow_dev.md

```diff
@@ -0,0 +1,22 @@
+Following steps are to help developer install and config ecflow workflow for realtime cycled parallel run
```

Comment:
Can you please use markdown syntax with itemized steps.
This reads like a text file.